cli/cmd/plugin/unmanaged-cluster/tanzu/tanzu.go

```diff
@@ -252,23 +254,26 @@ func (t *UnmanagedCluster) Deploy(scConfig *config.UnmanagedClusterConfig) (int,
 	blockForKappStatus(kappDeployment, kc)
 
 	// 6. Install package repositories
+	// Install and wait for core repo first
 	pkgClient := packages.NewClient(kcBytes)
 	log.Event(logger.EnvelopeEmoji, "Installing package repositories")
 	createdCoreRepo, err := createPackageRepo(pkgClient, tkgSysNamespace, tkgCoreRepoName, t.bom.GetTKRCoreRepoBundlePath())
 	if err != nil {
 		return ErrCorePackageRepoInstall, fmt.Errorf("failed to install core package repo. Error: %s", err.Error())
 	}
+	blockForRepoStatus(createdCoreRepo, pkgClient)
 
-	// Install the additional package repos
+	// Install and wait for the additional package repos
```

Comment:
Let's only block for reconciliation when a profile is specified.

ie: Most bootstraps, which won't set a profile, shouldn't wait for additional package repos to reconcile.